### DISCUSSION TITLE
I need help with converting this CloudFormation template to terraform modules/code

### DISCUSSION THREAD

**Discussion by aaditunni-unityinfotech** [DontShaveTheYak/cf2tf#156] [Q&A]
I tried doing it but ended up getting quite a lot of error which I have no idea about.
Here is the link to the template: https://github.com/aws-samples/aws-securityhub-remediations/blob/main/aws-ecr-continuouscompliance/cft/aws-ecr-continuouscompliance-v1.yaml

Link to the official blog: https://aws.amazon.com/blogs/containers/automating-image-compliance-for-amazon-eks-using-amazon-elastic-container-registry-and-aws-security-hub/

I would really appreciate if someone can help me with this.
Thank you !

**Comment by shadycuz** [comment]
Posting the error would help, but I will take a look at it this weekend.

**Reply by shadycuz** [reply]
I see you created a separate issue with your error but did not include the links to the templates like you did here.

**Reply by aaditunni-unityinfotech** [reply]
Oh, I totally forgot about that. I will edit the other one.
